.github/workflows/push-to-master.yml

```diff
@@ -115,7 +42,7 @@ jobs:
           python-version: ${{matrix.python-version}}
           channel-priority: strict
           use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-      - run: pip install -r requirements-dev.txt
+      - run: pip install -r requirements-dev.txt --use-deprecated=legacy-resolver
```

Comment:
This will be the only change.

Reply:
okay, revert the unnecessary changes then